pepper-apis/src/main/java/org/broadinstitute/ddp/route/PatchFormAnswersRoute.java

```diff
@@ -212,9 +212,13 @@ public Object handle(Request request, Response response) {
                             Set<Long> answerIds = answerDao.getAnswerSql()
                                     .findAnswerIdsByInstanceGuidAndQuestionId(instanceGuid, questionDto.getId());
                             if (answerIds.size() > 1) {
-                                String msg = "Question is already answered. Provide the answer guid to update.";
-                                LOG.info(msg);
-                                throw ResponseUtil.haltError(response, 409, new AnswerExistsError(msg, questionStableId));
+                                String errMsg = String.format(
+                                        "A question can have 1 and only 1 answer but found %d answers instead. "
+                                        + "Answer ids: %s",
+                                        answerIds.size(),
+                                        answerIds.toString()
```

Comment:
Looks like we'll return this message to clients in api response. Can we simplify this message a bit? For example, I don't think clients need to know the answer ids?